docs/semgrep-app/notifications.md

```diff
@@ -84,24 +84,24 @@ To enable MR comments:
 5. Click to **Add variable**, give the new variable the key `PAT` and use the token you copied in step 3 as the value. And then, select **mask variable** and **UNSELECT "protect variable"**.
 6. Update your .gitlab-ci.yml file with variable `GITLAB_TOKEN` and value `$PAT`. See the example code below for details:
 
-For example:
+#### Example
```

Comment:
@adamkvitek I'm a little adamant about this, since the example is about where $PAT should be and not with the purpose of providing a sample/example CI code to copy. Wdyt?

Reply:
@adamboros makes sense! 

We have two options: 

1. Make the header more specific to what you want to convey here. What about (feel free to change this one):
    ```md
    #### $PAT variable example
    ```

2. (Which I find much better.) I think we do not need a separate header for something that can be explained only by indentation. This will make clear that the example belongs to step number 6. Making a separate header is a bit too much here I think. See suggestion below the original:

Original:
```md
6. Update your .gitlab-ci.yml file with variable `GITLAB_TOKEN` and value `$PAT`. See the example code below for details:

#### Example
```yaml
semgrep:
  image: returntocorp/semgrep
  script:
    - semgrep ci
  rules:
  - if: $CI_MERGE_REQUEST_IID

  variables:
    SEMGREP_APP_TOKEN: $SEMGREP_APP_TOKEN
    GITLAB_TOKEN: $PAT
```
```

Suggesting
```md
6. Update your .gitlab-ci.yml file with variable `GITLAB_TOKEN` and value `$PAT`. See the example code below for details:
    ```yaml
    semgrep:
      image: returntocorp/semgrep
      script:
        - semgrep ci
      rules:
      - if: $CI_MERGE_REQUEST_IID

      variables:
        SEMGREP_APP_TOKEN: $SEMGREP_APP_TOKEN
        GITLAB_TOKEN: $PAT
    ```
```

Reply:
(Note) There is some weird issue with formatting here, please consider `Suggesting:` as not being a part of the code snippet above. @adamboros

Reply:
@adamkvitek I don't think this results in any indentation difference once rendered, am I missing something?

Reply:
@adamboros If you use VS code, there are plugins to build preview inside of the editor, which can help with some formatting things, although usually, you need to build the docs as well to make sure the output will work. ;-)

Reply:
Updating in the document.